### PR TITLE
Install `_stbt` library into global python directory

### PR DESCRIPTION
**Pull request by wmanley**
It means less fiddling with `$PYTHONPATH`.  It's called `_stbt` so that should be enough to prevent people from using it.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -86,6 +81,7 @@ INSTALL_CORE_FILES = \
     _stbt/x11.py \
     _stbt/xorg.conf.in \
     _stbt/xxhash.py \
+    stbt.conf \
```
Comment (drothlis) [inline Makefile]: I see, so now we install `stbt.conf` to `/etc/stbt/stbt.conf` *and* to `$libexecdir/stbt/stbt.conf` (the latter location used to be a modified version of `stbt.conf`, now it's just the identical file).

File: stbt/__init__.py [hunk 2]
```diff
@@ -15,8 +15,8 @@
 from contextlib import contextmanager
 
 try:
-    from stbt._vars import _libexecdir  # Generated by 'make install'. pylint:disable=import-error,no-name-in-module
-    sys.path.insert(0, _libexecdir + "/stbt")
+    from _stbt.vars import libexecdir  # Generated by 'make install'. pylint:disable=import-error,no-name-in-module
```
Comment (drothlis) [inline stbt/__init__.py]: This won't be able to find `_stbt` but that's fixed by the next commit.

File: Makefile [hunk 3]
```diff
@@ -135,6 +135,10 @@ install-core: all
 	done
 	printf "libexecdir = '%s'\nsysconfdir = '%s'\n" \
 	    "$(libexecdir)" "$(sysconfdir)" > $(DESTDIR)$(pythondir)/_stbt/vars.py
+	for filename in $(INSTALL_PYLIB_FILES); do \
+	    [ -x "$$filename" ] && mode=0755 || mode=0644; \
+	    $(INSTALL) -m $$mode $$filename $(DESTDIR)$(pythondir)/$$filename; \
+	done
 	chmod 0644 $(DESTDIR)$(pythondir)/_stbt/vars.py
```
Comment (drothlis) [inline Makefile]: The new code needs to go 1 line lower -- you've split up the code that deals with `_stbt/vars.py`.

File: Makefile [hunk 4]
```diff
@@ -135,6 +135,10 @@ install-core: all
 	done
 	printf "libexecdir = '%s'\nsysconfdir = '%s'\n" \
 	    "$(libexecdir)" "$(sysconfdir)" > $(DESTDIR)$(pythondir)/_stbt/vars.py
```
Comment (drothlis) [inline Makefile]: Do we still need to record `libexecdir`? Yes we do, for `_stbt/control.py::stbt_control_listen`, which imports `$libexecdir/stbt/stbt-control`, and which is used by `stbt record --control-recorder`.

Comment (wmanley) [reply Makefile]: The real fix there is to move (at least some of) the contents of `stbt-control` into a library under `_stbt`.  I couldn't be bothered to untangle it right now though.

File: Makefile (hunk 1 above)
Comment (wmanley) [reply Makefile]: The ideal would be to install `/etc/stbt/stbt.conf` as identical to `/usr/.../stbt.conf` but with all the variables commented out.  This way the options and their defaults are documented and it's clear which ones are being overridden.

**Pull request merged**